core/static/js/page.js

```diff
@@ -101,20 +101,12 @@
             var scale = 1 / all_coordinates["width"];
             
             $.each(words.split(" "), function(index, word) {
-                if (word!="") {
-                    var boxes = [];
-
-                    var coordinates = all_coordinates["coords"][word];
-                    if(coordinates !== undefined){
-                        $.each(coordinates, function(index, value) {
-                            addOverlay(viewer,
-                                       value[0]*scale,
-                                       value[1]*scale,
-                                       value[2]*scale,
-                                       value[3]*scale);
-                        });
-                    }
-                }
+                var boxes = [];
```

Comment:
have you tried any examples that would have run up against their `word!=""` check?

Reply:
If that check is necessary, we need to add it to highlight.js, since that script is responsible for the search result highlighting.  Actually, though, we can make that never possible by fixing the split.  BRB.

Reply:
New commit uses /\s+/ for the split in both files, so "" is not possible.

Reply:
oh duh, thanks

Reply:
thanks!